### PR TITLE
Orbit SDK intro edits

### PR DESCRIPTION
**Pull request by symbolpunk**
experimental edit that builds upon https://github.com/OffchainLabs/arbitrum-docs/pull/1132

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'How to Deploy a Token Bridge Using the Orbit SDK'
+title: 'Deploy a token bridge using the Orbit SDK'
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md:2]: sentence-casing ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: 'How to Configure a Node Using the Orbit SDK'
-sidebar_label: 'Node Config Generation'
+title: "How to configure your Orbit chain's node using the Orbit SDK"
+sidebar_label: "Configure your chain's node"
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md:3]: Sentence casing, title patterns ([ref](https://www.notion.so/arbitrum/How-to-775951f3d84c476cb6cba296a0a6f671?pvs=4))

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 description: 'How to configure a node using the Orbit SDK'
 author: GreatSoshiant
 sme: GreatSoshiant
 target_audience: 'Developers deploying and maintaining Orbit chains.'
-sidebar_position: 4
+content_type: how-to
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-preparing-node-config.md:8]: frontmatter patterns ([ref](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#71d2d7d4f24149a6b08b385f2581d90a))

File: arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md [hunk 4]
```diff
@@ -1,17 +1,67 @@
...
-author: Mehdi Salehi
-sme: Mehdi Salehi
-target_audience: 'Developers deploying and maintaining Orbit chains.'
-sidebar_position: 0
+title: 'Get started with the Arbitrum Orbit SDK'
+sidebar_label: 'Orbit SDK: Get started'
+description: 'Learn how to deploy and manage your Orbit chain with the Arbitrum Orbit SDK.'
+author: GreatSoshiant
+sme: GreatSoshiant
+user_story: As a current or prospective Orbit chain owner, I need to onboard into the Orbit SDK by understanding the available onboarding paths, and how to select the path that meets my needs.
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md:7]: frontmatter patterns ([ref](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#71d2d7d4f24149a6b08b385f2581d90a))

File: arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md [hunk 5]
```diff
@@ -1,17 +1,67 @@
...
+content_type: get-started
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md:8]: content type ([ref](https://www.notion.so/arbitrum/Get-started-bb8ab1b252b34f61a23e21ad2eed30b6?pvs=4))

File: arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md [hunk 6]
```diff
@@ -1,17 +1,67 @@
...
 ---
 
 import PublicPreviewBannerPartial from './partials/_orbit-public-preview-banner-partial.md';
 
 <PublicPreviewBannerPartial />
 
+The Arbitrum Orbit SDK lets you programmatically create and manage your own Orbit chain(s). Its capabilities include:
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md:15]: I've made an assumption: The purpose of this document is to onboard readers into the Orbit SDK as efficiently as possible. To that end, I've attempted to:

 - remove extraneous copy
 - optimize for actionable steps, decision-making, and skimmability

Comment (anegg0) [reply arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md:15]: I like your copy better than the original.

**Pull request merged**